Assets/Scripts/BranchSpawner.cs

```diff
@@ -9,6 +9,7 @@ public class BranchSpawner : MonoBehaviour
     private float _sap;
 
     public float maxSap;
+    public float branchSpawnLocationMagnitude = 9f;
```

Comment:
Can this be assigned in Unity instead of at the top of this file, since its public? Also, what do you think of changing the variable name to spawnLocationOffset? I think it more clearly states what the variable does.